Source/WebCore/PAL/pal/PALSwift.h

```diff
@@ -28,8 +28,39 @@
 #include <cstdint>
 #include <wtf/Vector.h>
 
+namespace Cpp {
+
 using VectorUInt8 = WTF::Vector<uint8_t>;
 using SpanConstUInt8 = std::span<const uint8_t>;
+using OptionalVectorUInt8 = std::optional<WTF::Vector<uint8_t>>;
+
+
+// FIXME: remove when swift support is available rdar://118026392
+inline OptionalVectorUInt8 toOptionalVectorUInt8(VectorUInt8 val)
+{
+    return val;
+}
+
+inline VectorUInt8 createVectorUInt8FromSpan(SpanConstUInt8 span)
+{
+    return VectorUInt8(span);
+}
+
+inline void appendSpanToVectorUInt8(const SpanConstUInt8& span, VectorUInt8& vec)
```

Comment:
I would call this append() and put the Vector argument first. It's just a style preference, but that's how most WebKit code works -- object first, argument second. 

Does Vector::append(std::span<U, Extent>) not work in Swift? If so, probably needs a Radar and comment. The purpose of the comment is to help explain the programming model. If we're doing something to work around a bug, saying so explicitly can help the reader.

Reply:
> Does Vector::append(std::span<U, Extent>) not work in Swift


Nope, cause it's templated.